cms/djangoapps/contentstore/views/user.py

```diff
@@ -71,14 +81,20 @@ def _manage_users(request, course_key):
         raise PermissionDenied()
 
     course_module = modulestore().get_course(course_key)
-    instructors = CourseInstructorRole(course_key).users_with_role()
+    instructors = set(CourseInstructorRole(course_key).users_with_role())
     # the page only lists staff and assumes they're a superset of instructors. Do a union to ensure.
     staff = set(CourseStaffRole(course_key).users_with_role()).union(instructors)
 
+    formatted_users = []
+    for user in instructors:
+        formatted_users.append(user_with_role(user, 'instructor'))
+    for user in staff - instructors:
+        formatted_users.append(user_with_role(user, 'staff'))
+
     return render_to_response('manage_users.html', {
         'context_course': course_module,
-        'staff': staff,
-        'instructors': instructors,
+        'transfer_ownership': request.user in instructors and len(instructors) == 1,
```

Comment:
This variable name confused me-- it isn't representing that transferring of ownership should/will happen, it's indicating that transferring of ownership can't happen and some text will be shown explaining the situation.

Perhaps "show_transfer_ownership_text" or something to that effect.